dask/bag/core.py

```diff
@@ -211,6 +211,27 @@ class Item(Base):
     _default_get = staticmethod(mpget)
     _finalize = staticmethod(finalize_item)
 
+    @staticmethod
+    def from_imperative(value):
```

Comment:
I added this here because the module level `from_imperative(values)` already has an explicit case for "single Value -> [value] -> Bag".  I personally feel that this case should simply return an `Item`, which would be less surprising, but a larger and user-visible change.